### PR TITLE
create folder signup

### PR DESCRIPTION
**Pull request by ravendra-prajapati-au2**

### CODE REVIEW

**Comment by samarth-keskar-au1** [comment]
@ravendra-prajapati-au2 
Issues =>
1=> The data is not being saved in database when the user sign up
2=> The Login is also not working.
3=> The input type of the passowrd should be "password".
4=> Do not hardcode the username and password in the POST Login route fetch username and password from database.
5=> You cannot send 2 responses in 1 route/ 1 request.

File: index.js [hunk 1]
```diff
@@ -29,24 +27,68 @@ app.use( "*", cloudinaryConfig );
...
+})
+
+app.post('/login', function (req, res) {
+
+    req.app.locals.db.collection('users').findOne({
+        username: req.body.log_username,
+        password: req.body.log_password
+    }, function (err, users) {
+        if (users!== 0) {
+            console.log("user exists");
```
Comment (samarth-keskar-au1) [inline index.js]: if the user is found create a session for that user.

File: index.js [hunk 2]
```diff
@@ -29,24 +27,68 @@ app.use( "*", cloudinaryConfig );
...
+    console.log("Record inserted Successfully");
+     
+    });
+    res.redirect('/index.html'); 
+})
+
+app.post('/login', function (req, res) {
+
+    req.app.locals.db.collection('users').findOne({
+        username: req.body.log_username,
```
Comment (samarth-keskar-au1) [inline index.js]: You are using req.body.log_username here and you have named your name attribute as 'username' only how is that going to work ?

Comment (ravendra-prajapati-au2) [reply index.js]: ok

File: index.js (hunk 1 above)
Comment (ravendra-prajapati-au2) [reply index.js]: ok

**Pull request merged**